http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java

```diff
@@ -107,6 +115,105 @@ public void testGrpc() {
         getApp().given().when().get("/api/grpc/trinity").then().statusCode(SC_OK).body(is("Hello trinity"));
     }
 
+    @Test
+    @DisplayName("GRPC reflection test - service count")
+    public void testReflection_serviceCount() {
+        GrpcReflectionResponse response = getApp().given().when().get("/api/grpc/reflection/service/info")
+                .then().statusCode(SC_OK).extract().response()
+                .jsonPath().getObject(".", GrpcReflectionResponse.class);
+
+        assertEquals(response.getServiceCount(), 3);
```

Comment:
@xtroitgeo in order to get correct validation error message, the format must be `assertEquals(expected, actual)`, please check out formal parameter names of the `assertEquals`.  When tests are failing, it gets confusing. Please correct it.

Reply:
Fixed